packages/next-swc/crates/core/Cargo.toml

```diff
@@ -29,7 +29,7 @@ next-binding = { git = "https://github.com/vercel/turbo.git", rev = "079505baf1c
 ] }
 
 [dev-dependencies]
-next-binding = { git = "https://github.com/vercel/turbo.git", rev = "079505baf1ce03707d28b6c0afb158dca601a11a", features = [
```

Comment:
I'd like to use an explicit `rev` hash here, so I don't need to review the git hash in `Cargo.lock` file.